include/RAJA/policy/sycl/launch.hpp

```diff
@@ -32,10 +32,13 @@ template <bool async>
 struct LaunchExecute<RAJA::sycl_launch_t<async, 0>> {
 
  //If the launch lambda is trivially copyable
-  template <typename BODY_IN,
+  template <typename BODY_IN, typename ReduceParams,
 	    typename std::enable_if<std::is_trivially_copyable<BODY_IN>{},bool>::type = true>
-  static resources::EventProxy<resources::Resource>
-  exec(RAJA::resources::Resource res, const LaunchParams &params, const char *kernel_name, BODY_IN &&body_in)
+    static concepts::enable_if_t<resources::EventProxy<resources::Resource>,
+                                 RAJA::expt::type_traits::is_ForallParamPack<ReduceParams>,
+                                 RAJA::expt::type_traits::is_ForallParamPack_empty<ReduceParams>>
```

Comment:
Do you need both?

Reply:
I was following the usage in forall and I would think so, since we need to consider the case that we are passing ForallParamPack and it is not empty which would lead to the other exec method. @mdavis36 what do you think?

Reply:
Either way if we do it it can be done separately.

Reply:
Checking `RAJA::expt::type_traits::is_ForallParamPack<ReduceParams>` is only necessary there exists another interface to `exec` that takes something after the lambda body that isn't a ForallParamPack. I can't immediately see if that's true here so you might be able to get away with just checking if it's empty and always assuming that `launch_reducers` is a `ForallParamPack` type. Removing it does mean that if exec is expanded in the future to take something besides a `ForallParamPack` then it will need to be added everywhere again.